Source/OctaneGUI/Controls/TextInput.cpp

```diff
@@ -890,6 +891,11 @@ void TextInput::AddText(const std::u32string& Contents)
     {
         return;
     }
+    
+    if (m_Text->Length() + Contents.length() > m_MaxCharacters)
```

Comment:
This should check to see if m_MaxCharacters > 0. If it is zero, the control should have no limit.

Reply:
Resolved in https://github.com/thnom/OctaneGUI/commit/079fb6b516a88c2a4334a7b7518609cf522d5bb9

Taken approach with if to do an _and_ but let me know if you would prefer it to check == 0 as its own block